client/stylesheets/_toCat.scss

```diff
@@ -1601,7 +1601,8 @@ HELP | ABOUT | FAQ | SAFETY | PRIVACY | CONTACT **/
     *******************/
 
     .oneDiv {
-        margin: 3rem 10% 2rem 10% !important;
+        width: 80%;
+        margin: 3rem auto 2rem auto !important;
```

Comment:
!important should not be used<br>Shorthand form for property `margin` should be written more concisely as `3rem auto 2rem` instead of `3rem auto 2rem auto`